packages/grafana-ui/src/components/DateTimePickers/DateTimePicker/DateTimePicker.tsx

```diff
@@ -389,4 +402,14 @@ const getStyles = (theme: GrafanaTheme2) => ({
     zIndex: theme.zIndex.modal,
     maxWidth: '280px',
   }),
+  clearIcon: css({
+    cursor: 'pointer',
+  }),
+  field: css({
+    marginBottom: 0,
+  }),
+  disabled: css({
```

Comment:
I tried that but then it breaks the layout, which looks weird when two or more pickers used next to each other. Not sure if it's a big deal tho 🤔 
![Screenshot 2024-05-23 at 14 09 51](https://github.com/grafana/grafana/assets/8878045/aeda8254-d126-4f4c-a204-e7dd5b366928)

Reply:
Ah yes because the suffix takes up space. I guess the solution would be to set a width for each of the DateTimePickers in the form if we want them all to have the same size. I'm assuming that's what we do with `Select` for instance to make sure the same thing doesn't happen?

Reply:
Yeah, that'd work but the picker doesn't have a configurable width and it's not 100% by default. Maybe I'll just hide the clear button even if it'll look a bit off.

Reply:
Good idea, lets do that and see if we get feedback. Then at least it would be consistent with how other clear buttons work 🙌